lib/cdo/contact_rollups/v2/contact_rollups.rb

```diff
@@ -13,5 +13,7 @@ def self.build_contact_rollups(log_collector)
     log_collector.time!('ContactRollupsPardotMemory.add_and_update_pardot_ids') {ContactRollupsPardotMemory.add_and_update_pardot_ids}
 
     # TODO: sync upstream step
+
+    log_collector.time!("ContactRollupsFinal.truncate_and_overwrite!") {ContactRollupsFinal.truncate_and_overwrite!}
```

Comment:
Shouldn't it be `truncate_and_overwrite` instead of `truncate_and_overwrite!`? You can test the pipeline by running ContactRollupsV2.build_contact_rollups in a Rails console.

Reply:
👍 done!